t/test-rest-api.php

```diff
@@ -319,6 +319,41 @@ function test_endpoint_get_entries() {
 
 	/**
 	 * Integration test
+	 * Test accessing the crud endpoint with an insert action
+	 */
+	function test_endpoint_crud_action() {
+
+		// Create an author and set as the current user
+		$author_id = $this->factory->user->create( array(
+			'role' => 'author',
+		) );
+		wp_set_current_user( $author_id );
+
+		// Create a post
+		$this->factory->post->create();
```

Comment:
During the test we never actually convert the post into a liveblog; should the insert action pass?